cloudslang-compiler/src/main/java/io/cloudslang/lang/compiler/SlangSource.java

```diff
@@ -55,12 +55,11 @@ private static String readFileToString(File file) throws IOException {
     }
 
     private static Charset getCharset() {
-        Charset charset = Charset.defaultCharset();
         String cslangEncoding = System.getProperty(SlangSystemPropertyConstant.CSLANG_ENCODING.getValue());
-        if (!StringUtils.isEmpty(cslangEncoding)) {
-            charset = Charset.forName(cslangEncoding);
+        if (StringUtils.isEmpty(cslangEncoding)) {
+            cslangEncoding = "UTF-8";
```

Comment:
I think we should define a const for `"UTF-8"` - maybe `SlangSystemPropertyConstant` enum is a good place - WDYT?